go.mod

```diff
@@ -53,6 +53,7 @@ require (
 	github.com/stretchr/testify v1.9.0
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 	github.com/zeebo/blake3 v0.2.3
+	go.uber.org/atomic v1.11.0
```

Comment:
Why the new dependency? Can't we use `sync/atomic` or `sync.Mutex`?

An `atomic.Duration` from `go.uber.org/atomic` is essentially just a `atomic.Int64` from the `sync` package?

Reply:
We can, but than every time comparing time.Duration and int64 we will have to convert types. 
So atomic.Duration requires less code, imho.

Reply:
It adds a dependency on a package to do this:

```go
c.fetchedPhaseShift.Store(resp.PhaseShift)
value := c.fetchedPhaseShift.Load()
```

instead of that:

```go
c.fetchedPhaseShift.Store(int64(resp.PhaseShift))
value := time.Duration(c.fetchedPhaseShift.Load())
```